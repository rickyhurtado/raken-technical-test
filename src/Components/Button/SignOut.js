import React, { Component } from 'react';

export default class SignOut extends Component {
  render() {
    return (
      <li>
        <button className="sign-out-btn">Sign Out</button>
      </li>
    );
  }
};
