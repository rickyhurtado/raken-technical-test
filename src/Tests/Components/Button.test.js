import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../../Components/Button';

it('Button.SignOut component should render', () => {
  const component = renderer.create(<Button.SignOut />);

  expect(component.toJSON()).toMatchSnapshot();
});
