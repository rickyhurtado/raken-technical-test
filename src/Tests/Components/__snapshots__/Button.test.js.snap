// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button.SignOut component should render 1`] = `
<li>
  <button
    className="sign-out-btn"
  >
    Sign Out
  </button>
</li>
`;
